### ISSUE TITLE
start testing with Go 1.13, switch to Actions for CI

### ISSUE THREAD

**Issue by mvdan** [gunk/gunk#267]
The tests crash spectacularly on 1.13 and master, for some reason:

```
$ go test ./...
2019/10/05 13:24:30 internal error: nil Pkg importing "errors" from "time"
FAIL    github.com/gunk/gunk    0.896s
```

**Comment by karel-3d** [comment]
should this be closed?

**Comment by favadi** [comment]
Should be closed with #304.
